### PR TITLE
Test tier updates for Spaces

### PR DESCRIPTION
**Pull request by rajivnathan**
Summary of changes:
- Adds print debug make target for local source testing: `deploy-e2e-local-and-print-local-debug`
- Completes test `TestUpdateNSTemplateTierWithSpaces`

Related Issue: https://issues.redhat.com/browse/CRT-1314

Related PR: https://github.com/codeready-toolchain/host-operator/pull/568

### CODE REVIEW

File: test/e2e/nstemplatetier_test.go [hunk 1]
```diff
@@ -240,48 +238,26 @@ func TestUpdateNSTemplateTierWithSpaces(t *testing.T) {
...
-	setupSpaces(t, awaitilities, "icecream", "icecreamlover%02d", memberAwait, count)
+	spaces := setupSpaces(t, awaitilities, "icecream", "icecreamlover%02d", memberAwait, count)
 
-	// verify icecream tier created successfully
-	_, err := hostAwait.WaitForNSTemplateTier("icecream", UntilNSTemplateTierStatusUpdates(1))
-	require.NoError(t, err)
+	verifyResourceUpdatesForSpaces(t, hostAwait, memberAwait, spaces, "icecream", "base", "base")
+
+	// t.Log("Sleeping for 1 minute")
+	// time.Sleep(time.Minute)
```
Comment (MatousJobanek) [inline test/e2e/nstemplatetier_test.go:246]: :ghost: 
```suggestion
```

**Review by MatousJobanek — APPROVED** [review]
Overall it looks good, I was just thinking if we could merge the space update code with the code in `TestUpdateNSTemplateTier` function. 

In addition, the function `TestUpdateNSTemplateTierWithSpaces` is commented with:
```
// TODO: this test should be removed once migration from MUR -> Spaces is completed.
```
which is technically not true anymore. We will need to support updates for both types of the Spaces:
* created for UserSignup
* created on its own for AppStudio case

**Review by xcoulon — APPROVED** [review]

File: test/e2e/nstemplatetier_test.go (hunk 1 above)
Comment (rajivnathan) [reply test/e2e/nstemplatetier_test.go:246]: Oops!

**Comment by rajivnathan** [comment]
@MatousJobanek thanks for the review! I removed the TODO comment. Also, I agree it makes sense to merge the Space update code into TestUpdateNSTemplateTier since we need to support both types of Spaces as you mentioned. Done in 04e2ecd10aa5304725ada9be156598747d37ba5e.

**Comment by rajivnathan** [comment]
@MatousJobanek all comments have been addressed. Can you please take another look when you have some time. Thanks!

**Review by MatousJobanek — APPROVED** [review]
Cool, thanks for merging the test functions :+1:

**Pull request merged**